<launch>
  <node name="image_proc" pkg="image_proc" type="image_proc" ns="/pylon_camera_node"/>
  <node name="bag" pkg="rosbag" type="play"
      args="-l $(find erfnet_ros)/bags/example_synced.bag" />
</launch>
